install/update_91_92.php

```diff
@@ -1908,6 +1908,33 @@ function update91to92() {
    $migration->addKey('glpi_tasktemplates', 'users_id_tech');
    $migration->addKey('glpi_tasktemplates', 'groups_id_tech');
 
+   if (!$DB->fieldExists('glpi_items_operatingsystems', 'is_deleted')) {
```

Comment:
just one point, should this update part be placed after operatingsystems stuff, like ~ l1220

Reply:
sure :)